movai_core_shared/consts.py

```diff
@@ -96,3 +96,13 @@
 
 # Domains
 INTERNAL_DOMAIN = "internal"
+
+#container_names
+INFLUXDB_HOST = "influxdb"
+MESSAGE_SERVER_HOST = "message-server"
+
+
+#inluxdb DB names:
+INFLUX_LOGS_DB_NAME = "logs"
+INFLUX_METRICS_DB_NAME = "metrics"
+INFLUXDB_DB_NAMES = [INFLUX_LOGS_DB_NAME, INFLUX_METRICS_DB_NAME]
```

Comment:
@engdorm I think we should use .env file here instead of consts.py because it can be changed in the future 
there is a nice python module specifically for that (dotenv) see use cases [here](https://dev.to/jakewitcher/using-env-files-for-environment-variables-in-python-applications-55a1#:~:text=that%20utilizes%20them.-,A%20.,potentially%20sensitive%20information%20at%20risk.)

what do you think both ? @engdorm @erezz-mov-ai

Reply:
I think those things should stay simple!